pkg/ui/console/client.go

```diff
@@ -30,9 +31,9 @@ type Client struct {
 }
 
 func CreateClient(ctx context.Context, configPath string, opts ...client.Option) (*Client, error) {
-	cfg, err := loadConfig(configPath)
-	if err != nil {
-		return nil, err
+	cfg, ok := loadConfig(configPath)
+	if !ok {
+		return nil, fmt.Errorf("")
```

Comment:
Add comment to explain why we're using empty error string? or maybe use a typed error, or even `console.ExitCodeError`

Reply:
I agree we should maybe define console error codes, this important to diff between an actual error or a failure for example fetch failed/drift scan returned unmanaged resources/policy fails. Maybe out of scope from this PR. we can open a new issue on that.

Reply:
@roneli console.ExitCodeError already exists (but it takes an `int`, not a special-typed exitcode or const) my point was to use it instead of an empty error message, since the error message is already handled in the caller as a diag.